test/SlowTests/Sharding/Issues/RavenDB_22200.cs

```diff
@@ -0,0 +1,134 @@
+
+                    Tuple<bool, DynamicJsonValue> alertRaised = null;
+                    await AssertWaitForValueAsync(async () =>
+                    {
+                        alertRaised = await notificationsQueue.TryDequeueAsync(TimeSpan.FromMinutes(1));
+
+                        if (alertRaised == null ||
+                            string.Compare(alertRaised.Item2["Type"].ToString(), NotificationType.AlertRaised.ToString(), StringComparison.OrdinalIgnoreCase) != 0 ||
+                            alertRaised.Item2[nameof(AlertRaised.Title)] == null ||
+                            string.Compare(alertRaised.Item2[nameof(AlertRaised.Title)].ToString(), "Resharding Delay Due to an Error",
+                                StringComparison.OrdinalIgnoreCase) != 0)
+                            return false;
+
+                        return true;
+                    }, true, 60_000);
+
+                    Assert.NotNull(alertRaised);
+                    var msg = alertRaised.Item2[nameof(AlertRaised.Message)]?.ToString();
+                    Assert.NotNull(msg);
+
+                    var expectedMsg = $"An error occurred while attempting to clean up bucket '{fromBucket}' from source shard '{fromShard}' [{db1.Name}].";
+                    Assert.Equal(expectedMsg, msg);
+                }
+            }
+        }
+
+        private sealed class StartManualReshardingOperation : IMaintenanceOperation
+        {
+            private readonly long _fromBucket;
+            private readonly long _toBucket;
+            private readonly int _toShard;
+
+            public StartManualReshardingOperation(long fromBucket, long toBucket, int toShard)
+            {
+                _fromBucket = fromBucket;
+                _toBucket = toBucket;
+                _toShard = toShard;
+            }
+
+            public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationContext context)
+            {
+                return new StartManualReshardingCommand(_fromBucket, _toBucket, _toShard);
+            }
+
+            private sealed class StartManualReshardingCommand : RavenCommand
+            {
+                private readonly long _fromBucket;
+                private readonly long _toBucket;
+                private readonly int _toShard;
+
+                public StartManualReshardingCommand(long fromBucket, long toBucket, int toShard)
+                {
+                    _fromBucket = fromBucket;
+                    _toBucket = toBucket;
+                    _toShard = toShard;
+                }
+
+                public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+                {
+                    url = $"{node.Url}/admin/resharding/start?fromBucket={_fromBucket}&toBucket={_toBucket}&toShard={_toShard}&database={node.Database}";
```

Comment:
shouldn't we encode the database name?

Reply:
in all other places where we use database names, we keep it as is